snowpack/src/build/import-resolver.ts

```diff
@@ -119,3 +120,37 @@ export function createImportResolver({fileLoc, config}: {fileLoc: string; config
     return false;
   };
 }
+
+/**
+ * Create a import glob resolver function, which converts any import globs relative to the given file at "fileLoc"
+ * to a local file. These will additionally be transformed by the regular import resolver, so they do not need
+ * to be finalized just yet
+ */
+export function createImportGlobResolver({fileLoc, config}: {fileLoc: string; config: SnowpackConfig}) {
+  return async function importGlobResolver(spec: string): Promise<string[]> {
```

Comment:
So this Windows bug is tricky! I think it has to do with modifying the `spec` string as you go. Basically, this is trying to serve 2 purposes, and getting tripped up between the two.

Its first purpose is keeping the original glob specifier, e.g. `import * from './**/*.js';`

But its second purpose is trying to also store a reference to an actual resolved file, which is hard to do until it’s actually looked up. And writing to the same reference is hard.